packages/editor-ui/src/components/SettingsLogStreaming/EventDestinationSettingsModal.ee.vue

```diff
@@ -22,338 +20,336 @@ import {
-			this.workflowsStore.removeNode(this.node);
-			this.ndvStore.activeNodeName = options.id ?? 'thisshouldnothappen';
-			this.workflowsStore.addNode(destinationToFakeINodeUi(options));
-			this.nodeParameters = options as INodeParameters;
-			this.logStreamingStore.items[this.destination.id].destination = options;
-		},
-		onTypeSelectInput(destinationType: MessageEventBusDestinationTypeNames) {
-			this.typeSelectValue = destinationType;
-		},
-		async onContinueAddClicked() {
-			let newDestination;
-			switch (this.typeSelectValue) {
-				case MessageEventBusDestinationTypeNames.syslog:
-					newDestination = Object.assign(deepCopy(defaultMessageEventBusDestinationSyslogOptions), {
-						id: this.destination.id,
-					});
-					break;
-				case MessageEventBusDestinationTypeNames.sentry:
-					newDestination = Object.assign(deepCopy(defaultMessageEventBusDestinationSentryOptions), {
-						id: this.destination.id,
-					});
-					break;
-				case MessageEventBusDestinationTypeNames.webhook:
-					newDestination = Object.assign(
-						deepCopy(defaultMessageEventBusDestinationWebhookOptions),
-						{ id: this.destination.id },
-					);
-					break;
 			}
+		}
+	});
+});
 
-			if (newDestination) {
-				this.headerLabel = newDestination?.label ?? this.headerLabel;
-				this.setupNode(newDestination);
-			}
-		},
-		valueChanged(parameterData: IUpdateInformation) {
-			this.unchanged = false;
-			this.testMessageSent = false;
-			const newValue: NodeParameterValue = parameterData.value as string | number;
-			const parameterPath = parameterData.name?.startsWith('parameters.')
-				? parameterData.name.split('.').slice(1).join('.')
-				: parameterData.name || '';
-
-			const nodeParameters = deepCopy(this.nodeParameters);
-
-			// Check if the path is supposed to change an array and if so get
-			// the needed data like path and index
-			const parameterPathArray = parameterPath.match(/(.*)\[(\d+)\]$/);
-
-			// Apply the new value
-			if (parameterData.value === undefined && parameterPathArray !== null) {
-				// Delete array item
-				const path = parameterPathArray[1] as keyof MessageEventBusDestinationOptions;
-				const index = parameterPathArray[2];
-				const data = get(nodeParameters, path);
-
-				if (Array.isArray(data)) {
-					data.splice(parseInt(index, 10), 1);
-					nodeParameters[path] = data as never;
-				}
-			} else {
-				if (newValue === undefined) {
-					unset(nodeParameters, parameterPath);
-				} else {
-					set(nodeParameters, parameterPath, newValue);
-				}
-			}
+function onInput() {
+	unchanged.value = false;
+	testMessageSent.value = false;
+}
+
+function onTabSelect(tab: string) {
+	activeTab.value = tab;
+}
+
+function onLabelChange(value: string) {
+	onInput();
+	headerLabel.value = value;
+	nodeParameters.value.label = value;
+}
+
+function setupNode(options: MessageEventBusDestinationOptions) {
+	workflowsStore.removeNode(node.value);
+	ndvStore.activeNodeName = options.id ?? 'thisshouldnothappen';
```

Comment:
No idea how this made into the code "thisshouldnothappen" 😆

Reply:
doesn't seem to be from this PR though. https://github.com/n8n-io/n8n/pull/4860/files#diff-4c588129b99621a76498f97fa57f76a20ebda61480ceac0d4ac36a1682090d76R361